src/SourceLocator/Type/FileIteratorSourceLocator.php

```diff
@@ -83,8 +84,8 @@ public function locateIdentifier(Reflector $reflector, Identifier $identifier):
      *
      * @throws InvalidFileLocation
      */
-    public function locateIdentifiersByType(Reflector $reflector, IdentifierType $identifierType): array
+    public function locateIdentifiersByType(Reflector $reflector, IdentifierType $identifierType): Generator
```

Comment:
The implementation of this class can be rolled back, except for the return type being `iterable`